src/dlr_tvm.cc

```
@@ -214,3 +223,34 @@ void TVMModel::UseCPUAffinity(bool use) {
     LOG(INFO) << "CPU Affinity is disabled";
   }
 }
+
+bool TVMModel::HasMetadata() const {
+  return this->metadata != nullptr;
+}
+
+const char* TVMModel::GetOutputName(const int index) const {
+  if (!this->HasMetadata()) {
+    LOG(FATAL) << "No metadata file was found!";
+  }
+  return this->metadata["Model"]["Outputs"][index]["name"].get<std::string>().c_str();
+}
+
+int TVMModel::GetOutputIndex(const char* name) const {
+  if (!this->HasMetadata()) {
+    LOG(FATAL) << "No metadata file was found!";
+  }
+  for (int i = 0; i < this->num_outputs_; i++) {
+    std::string name_str = this->metadata["Model"]["Outputs"][i]["name"];
+    if (name == name_str) {
+      return i;
+    }
+  } 
+  return -1;
+}
+
+void TVMModel::GetOuputByName(const char* name, float* out) {
+  int output_index = this->GetOutputIndex(name);
+  if (output_index >= 0) {
+    this->GetOutput(output_index, out);
+  }
```

It will silently do nothing if name is not found